content/updates/2020-09/1-flat-files-exports-with-labels.md

```diff
@@ -0,0 +1,32 @@
+---
+pim_announcement_img: ./img/Quick-export-new-UI_Export-with-labels.png
+pim_announcement_alt_img: "Flat files exports with labels"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# Export flat files with labels
+::: meta-data type="New" features="Flat files exports" available="early September" link-to-doc="../articles/exports.html"
+:::
+
+For your return from the summer holidays, we decided to tackle a widely shared issue, the impossibility for humans to read your flat files exports.
```

Comment:
The first paragraph of your news should be straight to the point and state what's new. Indeed, as this paragraph is used in the communication panel of the PIM, users can understand in a glimpse what is new, without having to click on the "Read more" button.